MeetMaryam/blog.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">   
+<link href="C:\Users\yonis\.vscode\School of Code\MeetMaryam\style.css" rel="stylesheet" type="text/css">
+
+<div class="header">
+    <h2>&#10024 Hello I'm Maryam! &#10024</h2>
+</div>
+
+<div class="row">
+    <div class="leftcolumn">
+        <div class="card">
+        <h2>Welcome to my profile</h2>
+        <img src="https://t3.ftcdn.net/jpg/01/76/98/40/360_F_176984023_8I82qQPmKn8TqNAZXIYMCSiwccoUiPBg.jpg" alt="Welcome sign"
+        width="500"
+        height="300">
+        <p>Hello I'm so glad that you have found my blog, I hope you find everything you're looking for.</p>
+    </div>
+    <div class="card">
+        <h2>New beginnings</h2>
+        <img src="https://newbeginning.org.uk/wp-content/uploads/2014/12/New-Beginnings-Support-Services1-1.jpg" alt="plant growing"
+        width="400"
+        height="200">
+        <img src="https://pbs.twimg.com/profile_images/1349310731062226950/gJy0xFBu_400x400.jpg" alt="Logo of SoC"
+        width="300"
+        height="200">
+        <p>This is the beginning of a new journey for me and just cant wait to get started. </p>
+    </div>
+</div>
+<div class="rightcolumn">
+    <div class= "card">
+        <h2>About Me</h2>
+        <img src="https://i.pinimg.com/736x/8c/57/4c/8c574cefd3d8a1d6086ef759eda77fb7--hijab-chic-hijab-style.jpg" alt="Hijabi woman"

```

Comment:
Well done for adding an alt tag, but I would say 'Hijabi woman' is a little impersonal. Get creative with your descriptions for people with accessibility needs.